### ISSUE TITLE
Choose a custom font for pages and posts

### ISSUE THREAD

**Issue by yechs** [yrccondor/mdx#54]
Can I choose a custom font for pages and posts?

Currently the theme seems to use 微软雅黑 as the default font, which has a bad support for English punctuation. (e.g. An apostrophe is displayed in full-width rather than half-width)

Or alternatively, is it possible to specify a different English font in parallel with 微软雅黑 as the Chinese font?

**Comment by yechs** [comment]
That's what 微软雅黑 looks like for English punctuation. The page is located [at here](https://blog.shuye.dev/sample-page/)

![image](https://user-images.githubusercontent.com/49149993/67360897-af381380-f599-11e9-8f4f-4aab90052318.png)

**Comment by yechs** [comment]
The alternative solution should be doable.

Check [Stack Overflow: How to use different font face for different languages?](https://stackoverflow.com/questions/49490011/how-to-use-different-font-face-for-different-languages)

**Comment by yrccondor** [comment]
It does a problem. But I think https://zenozeng.github.io/fonts.css/ may be a better solution. What's your opinion?

**Comment by yechs** [comment]
> 
> 
> It does a problem. But I think [zenozeng.github.io/fonts.css](https://zenozeng.github.io/fonts.css/) may be a better solution. What's your opinion?

That looks great! It's a much more elegant solution to our problem

**Comment by yechs** [comment]
```css
font-family: -apple-system, "Noto Sans", "Helvetica Neue", Helvetica, "Nimbus Sans L", Arial, "Liberation Sans", "PingFang SC", "Hiragino Sans GB", "Noto Sans CJK SC", "Source Han Sans SC", "Source Han Sans CN", "Microsoft YaHei", "Wenquanyi Micro Hei", "WenQuanYi Zen Hei", "ST Heiti", SimHei, "WenQuanYi Zen Hei Sharp", sans-serif; 
```

By placing English fonts before Chinese fonts, the English fonts will take precedence to render English characters, meanwhile leaving CJK characters to the Chinese fonts. 

Brilliant!

**Comment by yrccondor** [comment]
Fixed in 0e56ce3
